cli/cmd/app/logs/logs_test.go

```diff
@@ -0,0 +1,130 @@
+package logs
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"numerous/cli/internal/app"
+	"numerous/cli/test"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func dummyPrinter(entry app.AppDeployLogEntry) {}
+
+func TestLogs(t *testing.T) {
+	const slug = "organization-slug"
+	const appName = "app-name"
+	testError := errors.New("test error")
+
+	t.Run("given invalid slug then it returns error", func(t *testing.T) {
+		appDir := t.TempDir()
+		test.CopyDir(t, "../../../testdata/streamlit_app", appDir)
```

Comment:
does the first two lines have any impact on the test? If not, delete them 😄  this also goes for the next test

Reply:
Yeah, I'll remove those :smile:

Reply:
In 81eaf91